slushfile.js

```diff
@@ -240,13 +241,17 @@ function configRoxy() {
       'user=' + settings.marklogicAdminUser + '\n' +
       'password=' + settings.marklogicAdminPass + '\n';
 
-    fs.writeFileSync('deploy/local.properties', localProperties, {encoding: 'utf8'});
+    fs.writeFileSync('deploy/local.properties', localProperties, {
+      encoding: 'utf8'
+    });
```

Comment:
`{ encoding: 'utf8' }` is used in a few places, put in a var for denser code as well?